boxsdk/session/box_session.py

```diff
@@ -211,14 +173,38 @@ def _is_json_response(network_response):
         except ValueError:
             return False
 
-    def _retry_request_if_necessary(self, network_response, attempt_number, *args, **kwargs):
+    def _get_retry_after_time(self, attempt_number, retry_after_header):
         """
-        Retry a request for certain types of failure.
+        Get the amount of time to wait before retrying the API request.
+
+        If the Retry-After header is supplied, use it; otherwise, use exponential backoff
+        For 202 Accepted (thumbnail or file not ready) and 429 (too many requests), retry later, after a delay
+        specified by the Retry-After header.
+        For 5xx Server Error, retry later, after a delay; use exponential backoff to determine the delay.
+
```

Comment:
The doctoring talks about 202's, 500's, etc. But the code below could care less. The code is purely based on retry_after_header or not. 

It seems the doctoring, talking about status_codes, is really documenting some higher level logic elsewhere in the code?